streams/examples/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedJob.java

```diff
@@ -30,22 +31,35 @@
 import org.apache.kafka.streams.kstream.KStream;
 import org.apache.kafka.streams.kstream.KTable;
 import org.apache.kafka.streams.kstream.KeyValueMapper;
+import org.apache.kafka.streams.kstream.ValueJoiner;
 import org.apache.kafka.streams.kstream.Windowed;
 import org.apache.kafka.streams.StreamsConfig;
 
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Properties;
 
+/**
+ * Implements the "PageView Stats" program that joins the pageview stream with
+ * the user profile table to count the number of pageview per user region.
+ * Assumes the input stream is named "streams-pageview-input".
```

Comment:
Should we also clarify the format of the input stream (the name is pretty obvious to see in the example code, it's a bit harder though to understand the data format of the stream)?